net/nimble/host/include/host/ble_store.h

```diff
@@ -131,6 +137,40 @@ union ble_store_value {
     struct ble_store_value_cccd cccd;
 };
 
+struct ble_store_status_event {
+    /**
+     * The type of object that failed to persist; one of the
+     * BLE_STORE_OBJ_TYPE_[...] codes.
+     */
+    int obj_type;
+
+    /**
+     * The type of event being reported; one of the BLE_STORE_EVENT_TYPE_[...]
+     * codes.
+     */
+    int event_code;
+
+    /**
+     * Additional data related to the event; the valid field is inferred from
+     * the obj_type,event_code pair.
+     */
+    union {
+        /**
+         * The record that failed to be written.  Valid for the following event
+         * types:
+         *     o BLE_STORE_EVENT_OVERFLOW
```

Comment:
Since a key for union data is pair (obj_type, event_code pair) could we extend comment here and say that value is valid for event type "BLE_STORE_EVENT_OVERFLOW" and all possible obj_type.

Similar we could do for description to conn_handle, but int this case we have only couple of obj_types valid.